kludges.js

```diff
@@ -375,6 +373,12 @@ function restoreMethod(obj, name) {
 }
 
 var signals;
+var swipeTrackers = [
+    Main?.overview?._swipeTracker,
+    Main?.overview?._overview?._controls?._workspacesDisplay?._swipeTracker,
+    Main?.wm?._swipeTracker,
+    Main?.wm?._workspaceAnimation?._swipeTracker
```

Comment:
I think it would be good to comment which of these correspond to which gnome versions. Makes it easier to drop support for old versions when we're ready.

Reply:
Thanks @smichel17, have implemented these changes (see latest commit).